src/common/utils.ts

```diff
@@ -105,20 +106,32 @@ function xrpToDrops(xrp: BigNumber.Value): string {
     .toString(10)
 }
 
-function toRippledAmount(amount: Amount): RippledAmount {
+function toRippledAmount(amount: Amount | RippledAmount): RippledAmount {
+  if (typeof amount === 'string')
+    return amount;
+
   if (amount.currency === 'XRP') {
     return xrpToDrops(amount.value)
   }
   if (amount.currency === 'drops') {
     return amount.value
   }
+
+  let issuer = amount.counterparty || amount.issuer
+
+  let tag
+
+  try {
+    ({classicAddress: issuer, tag} = xAddressToClassicAddress(issuer))
+  } catch (e) { /* not an X-address */ }
```

Comment:
Nice inline clarification :) 

(Does the xAddressToClassicAddress throw a more specific error message which would let the catch block explain the issue directly? Ex. catch(NotAnXAddressException) )

Reply:
I believe this will just throw a generic `new Error`...

There's a valid code path (w/ a classic Address) where that throws and it's not an error, so I dont think we'd need to explain this issue to the client.